templates/nas-grp-base.html.twig

```diff
@@ -89,7 +89,7 @@
                                 <img src="{{ asset('build/images/nas-grp-logo.png') }}" alt="logo"
                                      style="width: 233px; margin-top: 5px" class="mw-100">
                             </a>
-                            <a class="navbar-brand" href="https://data.gulfresearchinitiative.org/" target="_blank">
+                            <a class="navbar-brand" href="https://griidc.org" target="_blank">
```

Comment:
Similarly, should this be grp.griiidc.org here? Check with PO.

Reply:
same as before, this is correct.